src/bindings/build.py

```diff
@@ -34,6 +34,17 @@
     with open(header, "r") as hfile:
         ffi.cdef(hfile.read())
 
+# we need to easily control a few values for Windows builds
+source = []
+if os.getenv("PYNACL_SODIUM_STATIC") is not None:
+    source.append("#define SODIUM_STATIC")
+
+source.append("#include <sodium.h>")
+
+if os.getenv("PYNACL_SODIUM_LIBRARY_NAME") is not None:
+    libraries = [os.getenv("PYNACL_SODIUM_LIBRARY_NAME")]
```

Comment:
Since there's no suffix difference for static vs dynamic vs link time compile optimization library types Windows people appear to sometimes totally rename their libraries. We saw this with the slproweb OpenSSL library names before we switched to our own for cryptography. In this case there are vcxproj files (at least in the git tree, they don't appear to actually ship in the tarball) for the various VS201X releases that probably do give the "libsodium" name by default.